config/config.go

```diff
@@ -38,14 +41,23 @@ type Config struct {
 	FlagEE      bool
 	TestFlag    bool
 	Enabled     string `json:"enabled"`
+
+	// Service account configuration
+	ID         string `json:"uid"`
+	SecretPath string `json:"secret_path"`
+	Secret     string
```

Comment:
nit: why store this as a `string` vs `[]byte`? you end up converting it when you read it from the file and then when you want to use it. seems easier to keep it as a slice